README.md

```diff
@@ -34,7 +35,7 @@ Hey there, fellow developer !!!... I'm happy to see you are interested in contri
 > [!IMPORTANT]
 > Make sure to check the [CONTRIBUTING.md](https://github.com/Grow-with-Open-Source/Javascript-Projects/blob/main/CONTRIBUTING.md "goto CONTRIBUTING.md") to understand the rules, file structure and step-by-step guide for contribution.
 
-- **Step 1:** Make sure you have required tools within you local machine *(like *git, vs code, node.js and so on)*.
+- **Step 1:** Make sure you have the required tools within you local machine *(like *git, vs code, node.js and so on)*.
```

Comment:
Please remove the _the_ from the sentence from line no.38 of the `README.md` file.
```suggestion
- **Step 1:** Make sure you have required tools within you local machine *(like *git, vs code, node.js and so on)*.
```